src/Products/PloneMeeting/browser/views.py

```diff
@@ -1416,6 +1416,26 @@ def printMeetingDate(self, returnDateTime=False, noMeetingMarker='-'):
         else:
             return noMeetingMarker
 
+    def print_preferred_meeting_date(self, returnDateTime=False, noMeetingMarker='-'):
+        """
+        Print preferred meeting date, manage fact that item has no preferred meeting date
+        :param returnDateTime if True, returns the preferred meeting date DateTime, otherwise it returns the
+        meeting title containing formatted date.
+        :param noMeetingMarker is returned when there is no preferredMeeting.
+        :return: Preferred meeting date
+        """
+        preferred_meeting_uid = self.context.getPreferredMeeting()
+        if preferred_meeting_uid == 'whatever':  # no need to do a query
+            return noMeetingMarker
+        catalog = api.portal.get_tool('portal_catalog')
+        brains = catalog.unrestrictedSearchResults(UID=preferred_meeting_uid)
+        if len(brains) == 1:  # catalog query is expected to have only one brain
+            if returnDateTime:
+                return brains[0].getObject().getDate()
+            tool = api.portal.get_tool('portal_plonemeeting')
+            return tool.formatMeetingDate(brains[0])
+        return noMeetingMarker  # catalog query was empty
+
     def print_in_and_out_attendees(
             self,
             in_and_out_types=[],
```

Comment:
Could you adapt getMeetingDate the same wahy (unrestricted) and propose all this in this PR?  then add a changelog too.  Remove printMeetingDate and rename it to print_meeting_date, check if that does not break already migrated cities...